app/models/game_night.rb

```diff
@@ -10,19 +10,23 @@ class GameNight < ApplicationRecord
   validates :number_of_games, presence: true
   validates :name, presence: true
 
-  def attendees
-    # I welcome optimizations here. But this took a while to
-    # realize that they needed to be separate.
+  def confirmed_attendees
     User
       .joins(:invitations)
-      .where(invitations: { game_night_id: id }) +
+      .where(invitations: { game_night_id: id, status: 'accepted' }) +
       User
       .joins(:game_nights)
       .where(game_nights: { id: id })
   end
 
+  def pending_attendees
+    User
+      .joins(:invitations)
+      .where(invitations: { game_night_id: id, status: 'pending' })
+  end
+
   def games_to_play
-    users = attendees.map { |user| user.id }
+    users = confirmed_attendees.map { |user| user.id }
```

Comment:
This is not a problem right now, but if there are no confirmed attendees, the list of games will just be the hosts until people say yes. Is that what you intended? I can go both ways here.